src/apps/main/auth/handlers.ts

```
@@ -48,7 +48,14 @@ export async function checkIfUserIsLoggedIn() {
 
   await checkUserData();
   encryptToken();
-  await createTokenSchedule();
+
+  try {
+    await createTokenSchedule();
+  } catch (exc) {
+    if (exc instanceof RefreshError) return;
+    else throw exc;
```

We don't want to to throw an exception in case of a refresh token error, we just want to omit the `emitUserLoggedIn`.